### ISSUE TITLE
Enabling 'Crop to page size' crops the images to nothing

### ISSUE THREAD

**Issue by xM8WVqaG** [cyanfish/naps2#142]
**Describe the bug**

After enabling "Crop to page size" the output is not unusable.

* The thumbnails and preview are broken. They look kinda like FFT-plots of the image(?) although I expect that might not be relevant.
* Saving the output creates 1x1 images and plain PDFs.

  ```text
  $ file crop-to-page-size.1.jpg
  crop-to-page-size.1.jpg: JPEG image data, JFIF standard 1.01, aspect ratio, density 1x1, segment length 16, baseline, precision 8, 1x1, components 3
  ```

**To Reproduce**

Under "Profile" > "Advanced" > "Compatibility", enable "Crop to page size".

![image](https://user-images.githubusercontent.com/44505331/235486519-dc34097f-e2db-43c8-bab9-fb8fc8a38f6e.png)

The two images on the left are scanned in colour. The two on the right are the same document in greyscale.

![image](https://user-images.githubusercontent.com/44505331/235486610-17f85523-8562-4fc7-b76f-1b98de32c659.png)

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04

```text
NAPS2 - Not Another PDF Scanner - Scan documents to PDF and more, as simply as
possible.

          ID: com.naps2.Naps2
         Ref: app/com.naps2.Naps2/x86_64/beta
        Arch: x86_64
      Branch: beta
     Version: 7.0b8
     License: GPL-2.0-or-later
      Origin: naps2-origin
  Collection: 
Installation: user
   Installed: 72.7 MB
     Runtime: org.freedesktop.Platform/x86_64/22.08
         Sdk: org.freedesktop.Sdk/x86_64/22.08

      Commit: 27e9023098c424f3aca01b33100722e169b76970db97671f4c83e1356ecedb1b
      Parent: 61dca37fe0ff53c2e876a59e6bca039366523d7cacc00e08c793cef396fa774b
     Subject: Export com.naps2.Naps2
        Date: 2023-04-22 22:54:49 +0000
```

**Additional context**

**Comment by christianclavet** [comment]
What was your paper size defined in the profile when you've done the test and what size was the real paper? Never user this feature before.

**Comment by xM8WVqaG** [comment]
I just checked - this was the last thing I did with NAPS so the profile was still configured - the paper size was "US Legal" and the paper in real life was A4 (everything I have is A4). Although I have another issue where all NAPS scans I make truncate to "US Letter".

I haven't used this feature via SANE before, but I have used a similar feature on Windows (probably) via TWAIN. It's quite useful for my scanner because it has a feeder with quite a wide slot so the paper guides don't keep paper perfectly straight when there is only one or two sheets.
